benchmark/buffers/buffer-iterate.js

```diff
@@ -4,8 +4,8 @@ const common = require('../common.js');
 const assert = require('assert');
 
 const bench = common.createBenchmark(main, {
-  size: [16, 512, 1024, 4096, 16386],
-  type: ['fast', 'slow'],
+  size: [512, 4096, 16386],
+  type: ['fast'],
```

Comment:
Similar to above: the `slow` part is mostly identical to the `fast` part.

I removed the super tiny buffer that should not have any implication on real applications and I removed a middle case since it's a linear increase.